### DISCUSSION TITLE
addReproduciblePeakSet function, output directory issue.

### DISCUSSION THREAD

**Discussion by Tomoliverberry** [GreenleafLab/ArchR#970] [Questions / Documentation]
Hi,
I'm having an issue using the reproducible peak set function. I am using the tile matrix method.
Having looked at the code that governs this function I notice that the outdir is specified with: 
_outDir <- file.path(getOutputDirectory(ArchRProj), "PeakCalls")_
I believe the issue is the slashes "/" present in the output directory, and I have included my code and errors.

I'm unsure if I want the directory to use forward or backward slashes or if I can edit the metadata outputDirectory value of my project to fix this issue.

Any help would be greatly appreciated. Thank you.

![image](https://user-images.githubusercontent.com/86775997/128912248-aa7f5d6f-e726-49b5-bb8b-226c6a4975d2.png)

**Comment by rcorces** [comment]
Sorry - ArchR isnt supported on Windows and we dont have bandwidth to help troubleshoot errors on Windows.
